admin/src/upload.rs

```diff
@@ -131,12 +132,19 @@ async fn upload(
     let headers: HeaderMap = headers
         .iter()
         .map(|s| {
-            let mut parts = s.splitn(2, ':');
-            let key = parts.next().unwrap().trim();
-            let value = parts.next().unwrap().trim();
-            (key.parse().unwrap(), value.parse().unwrap())
+            let Some((key, value)) = s.split_once(':') else {
```

Comment:
New syntax? I haven't seen this.

Reply:
kind of, learned about that just recently … https://doc.rust-lang.org/rust-by-example/flow_control/let_else.html